src/mitol/payment_gateway/api.py

```diff
@@ -453,7 +453,7 @@ def prepare_checkout(
 
                 formatted_merchant_fields[f"merchant_defined_data{idx}"] = field_data
 
-        consumer_id = hashlib.sha256(order.username.encode("ascii")).hexdigest()
+        consumer_id = hashlib.sha256(order.username.encode("utf-8")).hexdigest()
```

Comment:
The default encoding for `.encode` is `utf-8` but I mentioned it specifically in the function arg for more readability.